RecoTracker/LST/plugins/alpaka/LSTProducer.cc

```diff
@@ -36,21 +39,44 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       auto const& pixelSeeds = event.get(lstPixelSeedInputToken_);
       auto const& phase2OTHits = event.get(lstPhase2OTHitsInputToken_);
 
-      //sdl_.run(ctx.queue(), pixelSeeds, phase2OTHits);
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Maybe remove these `ifdef`, given that (for now) we specify `<flags ALPAKA_BACKENDS="cuda"/>` in the BuildFile?